src/wp-includes/class-wp-theme-json-resolver.php

```diff
@@ -403,6 +411,13 @@ public static function get_user_data_from_wp_global_styles( $theme, $create_post
 		if ( ! $theme instanceof WP_Theme ) {
 			$theme = wp_get_theme();
 		}
+
+		// Bail early if the theme does not support a theme.json. Since WP_Theme_JSON_Resolver::theme_has_support()
+		// only supports the active theme, the extra condition for whether $theme is the active theme is present here.
```

Comment:
Nits:
* multiline comment formatting
* I think we target commment lenghts of 80chars, so I've reformatted the lines too.

```suggestion
		/*
		 * Bail early if the theme does not support a theme.json.
		 *
		 * Since WP_Theme_JSON_Resolver::theme_has_support() only supports the active
		 * theme, the extra condition for whether $theme is the active theme is
		 * present here.
		 */
```